services/security/authentication/casigned/casigned.go

```diff
@@ -37,6 +44,47 @@ func (c *client) Login(ctx context.Context, group string, identity *security.Ide
 	return &response.Token, nil
 }
 
+func RenewRoutine(group, server string) {
+	renewalAttempt := 0
+	// Waiting for a few seconds to avoid spamming short-lived sdk user
+	time.Sleep(time.Second * 5)
```

Comment:
What is a "short-lived" sdk user?  
What is the sleep() trying to do?

Reply:
This is to avoid spamming the sdk client like mocctl

Reply:
So, you are avoiding renewing the certificate for mocctl?
is the expectation that mocctl will finish the command in 5 seconds and will thus exit the program?

Reply:
yes, I assume the login would take tell than that.

Reply:
what if the mocctl command runs for more than 5 seconds?  E.g. when you create a vm it can take more than 5 seconds.  Or if we are running on a overloaded computer?
Waiting 5 seconds is not a gaurantee.  It would be better to have the client pass in something that says it does not need to renew.

Reply:
This is only for the login api. VM creation would not invoke this. Yes, adding the parameter for renew would be better. I'll take it up next PR.

Reply:
ok.